projects/js-packages/components/components/scan-report/index.tsx

```diff
@@ -216,7 +216,10 @@ export default function ScanReport( { dataSource, data, onChangeSelection } ): J
 	 *
 	 * @see https://developer.wordpress.org/block-editor/reference-guides/packages/packages-dataviews/#getitemid-function
 	 */
-	const getItemId = useCallback( ( item: ScanReportExtension ) => item.id.toString(), [] );
+	const getItemId = useCallback(
+		( item: ScanReportExtension ) => `${ item.type }_${ item.slug }_${ item.version }`,
```

Comment:
Are we certain every item will have a type, slug and version? IIRC the `slug` for `core` is `null` and we don't currently appear to be supply a `slug` or `version` property for the files object we pass. Storybook runs fine so it doesn't really appear to matter, but perhaps thats only because `getItemId` is currently run given our setup.

Reply:
> perhaps wordpress_null_6.7.1 or files_null_null is fine.

This was/is my thought, as the generated IDs are still unique, specific to the item, and accurate.